csharp/sources/Valkey.Glide/Abstract/IServer.cs

```diff
@@ -100,7 +100,7 @@ public interface IServer
     Task<TimeSpan> PingAsync(CommandFlags flags = CommandFlags.None);
 
     /// <summary>
-    /// Pings the server.
+    /// This command is often used to test if a connection is still alive, or to measure latency.
```

Comment:
Why are we using a different description here vs what is used for IServerManagementClusterCommands?